### ISSUE TITLE
Get rid of yarn v1 for good! 

### ISSUE THREAD

**Issue by landsman** [yarnpkg/website#1202]
Guys, I really hate this day-to-day scenario when Google returns links to https://classic.yarnpkg.com/* as the first result. It's so annoying!

Could you please disable indexing for this website and, for God's sake, get rid of it permanently? 🙏

Please promote the latest version and place the history as a secondary option, somehow hidden below, like every other tool and service.

Is there any way I can help? 🙏

**Comment by landsman** [comment]
A few examples:

![Screenshot 2023-08-03 at 15 38 58](https://github.com/yarnpkg/website/assets/3184228/d402a18b-08e2-4a0e-9030-8db0645b28b1)

![Screenshot 2023-08-03 at 15 39 34](https://github.com/yarnpkg/website/assets/3184228/7cf2cbc9-2370-4989-962d-8b59982badc7)

![Screenshot 2023-08-03 at 15 39 54](https://github.com/yarnpkg/website/assets/3184228/3d8c71d8-95af-40ea-af26-0bfdb085334c)

It's time...

**Comment by landsman** [comment]
Related: https://github.com/yarnpkg/yarn/issues/8973

**Comment by arcanis** [comment]
Yes, I think removing the index is fine. We're actually wrapping up the [new website](https://yarn-v4-wip-website-dont-bookmark.netlify.app/), so we can do that once it's deployed (a couple of weeks).

**Comment by tspearconquest** [comment]
I'd also like to take a second to point out that the `node` docker image still pulls yarn 1.22.19. If anyone here has a connection with the team who manages the docker library where that image is built from, can you please ask them to update the images to use a supported yarn version?

**Comment by Daniel15** [comment]
Doesn't this mean that the old site has better SEO than the new site? 🤔

> get rid of it permanently?

If we want to remove the old site, all the old URLs should 301 (permanent redirect) to the correct page on the new site, for all languages the old site supported, such that none of the URLs indexed in Google become 404 errors.

> I'd also like to take a second to point out that the node docker image still pulls yarn 1.22.19.

As far as I can remember, Yarn v2 and newer are not designed to be installed globally in the same way that Yarn v1 was, which is why the Docker image, Debian package, etc are all still using Yarn v1. You can still have Yarn v1 installed globally but use Yarn v2/v3/... on a per-project basis.
